### PR TITLE
Add mini Big Sur guide

### PR DESCRIPTION
**Pull request by khronokernel**

### CODE REVIEW

File: extras/big-sur/fusion.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+Then, set the IMAGE variable to the name of the installer you are installing.  The example defines the image for Big Sur.
+
+```bash
+export IMAGE="Install macOS Beta"
+```
+
+Next, create an empty 16GB image to host the media.
+
+```bash
+makefile 16g "${IMAGE}.img"
```
Comment (martinxxd) [inline extras/big-sur/fusion.md:26]: Should be `mkfile 16g "${IMAGE}.img"`

Comment (dhinakg) [reply extras/big-sur/fusion.md]: Thanks!

**Comment by dhinakg** [comment]
Things remaining:
- [ ] Add AMD not supported note
- [ ] Finish USB passthrough for VMware
- [ ] Make sure VBox guide works on Windows/Linux

Things for a later PR:
- [ ] Workstation guide

**Comment by dhinakg** [comment]
Well OC has experimental support for KC now, gonna rewrite to use that

**Comment by zhigang1992** [comment]
IMHO this PR is very valuable to be merged now.

Even though it's still in beta, but these are the most detailed / well written / up to date doc there is right now on this subject.

Without this, we'd wasting hours searching and failed and not fully understanding what's going on.

**Pull request merged**